_episodes/24.md

```diff
@@ -0,0 +1,413 @@
+     time: 3320
+   - title: "Demo"
+     desc: "Running the Trino charts with kubectl"
+     time: 3462
+---
+
+This is the first episode in a series where we cover the basics and just enough
+advanced Kubernetes features and information to understand how to deploy Trino 
+on Kubernetes.
+
+## Concept of the week: K8s architecture: Containers, Pods, and kubelets
+
+For this concept of the week, we want to provide you a minimalistic overview of
+what you need to know for Kubernetes (also abbreviated k8s) to deploy a simple 
+Kubernetes cluster. 
+
+1. **Why Kubernetes?** Kubernetes is a container orchestration platform that allows
+   you to indicate how to manage containers declaritively using yaml 
+   configuration files. This definition can be tricky to understand if you don't
+   have proper context. To make sure nobody is left behind, it is useful to 
+   cover what containers are.
+    
+   - The traditional way to deploy an application, is to take the compiled 
+   binary of that application and run it directly on computer hardware that has
+   an operating system to run the application on it. This works, but had a lot
+   of dependency on the underlying hardware an operating system to be functional
+   and required multiple applications to share the same resources. If one of the
+   applications fails and cause any of the shared resources to crash, it could
+   affect all applications to fail on that machine. 
+    
+   - To remove these dependencies, engineers created virtual machines (VMs) by 
+   using a VM manager called the hypervisor that could run virtual environments 
+   that hosted other operating systems. This was a big step forward as now each 
+   application could be isolated, but it came at a great cost. Each virtual machine
+   hosted an entire operating system and was resource intensive and slow.
+
+   - Containers are the final type of deployment. Containers enable a logical
+   isolation of resources while still physically running on shared resources. 
+   All resources created in the hardware and operating systems exist on the host
+   system. The isolation restricts any interference from other processes. 
+   Containers achieve the goals of virtualization without sacrificing much 
+   performance or efficiency. 
+   
+
+   <p align="center">
+    <img align="center" width="75%" height="100%" src="/assets/episode/24/container-evolution.svg"/><br/>
+    Source: https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/
+   </p>
+   
+   - Containerization simplified a trend in service architecture called 
+   microservices. Microservices deploy loosely coupled and modular applications
+   rather than all-encompassing monolithic applications. With containers, these
+   applications can be deployed and scale up quickly across various virtual and
+   physical machines without affecting applications on the same machine. This
+   was great, but new problems emerged, such as, monitoring the health of
```

Comment:
is great, but results in new complexities. Example are the need for new approaches to ..